corehq/util/view_utils.py

```diff
@@ -107,5 +107,18 @@ def get_request():
     return global_request.get_request()
 
 
+def reverse(viewname, params=None, absolute=False, **kwargs):
```

Comment:
Because this is backwards-compatible, I think it's fine.  If you need to pass parameters, you can just swap out the import statement without worrying about breaking the other usages.